tutorials/preprocessing/plot_55_setting_eeg_reference.py

```diff
@@ -0,0 +1,136 @@
+# -*- coding: utf-8 -*-
+"""
+.. _tut-set-eeg-ref:
+
+Setting the EEG reference
+=========================
+
+This tutorial describes how to set an EEG reference in MNE-Python.
+
+.. contents:: Page contents
+   :local:
+   :depth: 2
+
+As usual we'll start by importing the modules we need, loading some
+:ref:`example data <sample-dataset>`, and cropping it to save memory:
+"""
+
+import os
+import mne
+
+sample_data_folder = mne.datasets.sample.data_path()
+sample_data_raw_file = os.path.join(sample_data_folder, 'MEG', 'sample',
+                                    'sample_audvis_raw.fif')
+raw = mne.io.read_raw_fif(sample_data_raw_file, verbose=False)
+raw.crop(tmax=60).load_data()
+
+###############################################################################
+# Background
+# ^^^^^^^^^^
+#
+# Typically, one of the first steps in processing EEG recordings is to subtract
+# a *reference signal* from each channel. Conceptually, the reference signal
+# represents environmental or equipment noise that affects all sensors
+# approximately equally, and by subtracting it from the signal at each
+# electrode, what is left will be a less noisy representation of brain activity
+# than the original raw signal.
```

Comment:
Choosing a proper reference has indeed a big effect on the noise levels. However, I would not start the story this way, otherwise people may be led to believe that EEG can exist without a reference.

EEG measures voltage. Voltage is the *difference* in electric potential between two points. Therefore, EEG *always* has a reference. `set_eeg_reference` simulates the signal as if another channel were the reference channel during the recording (or a 'virtual' channel that is the average of all channels, computed using the current reference).